src/main/java/com/google/api/generator/spring/composer/SpringAutoConfigClassComposer.java

```diff
@@ -401,37 +403,84 @@ private static List<AnnotationNode> createClassAnnotations(
     // @ConditionalOnProperty(value = "spring.cloud.gcp.language.enabled", matchIfMissing = true)
     // @EnableConfigurationProperties(LanguageProperties.class)
 
-    // TODO: AnnotationNode description only accepts String for now. need to extend to params
-    // and classes.
+    AssignmentExpr valueStringAssignmentExpr =
+        AssignmentExpr.builder()
+            .setVariableExpr(
+                VariableExpr.withVariable(
+                    Variable.builder().setName("value").setType(TypeNode.STRING).build()))
+            .setValueExpr(
+                ValueExpr.withValue(
+                    StringObjectValue.withValue(
+                        Utils.springPropertyPrefix(libName, service.name()) + ".enabled")))
+            .build();
+    AssignmentExpr matchIfMissingAssignmentExpr =
+        AssignmentExpr.builder()
+            .setVariableExpr(
+                VariableExpr.withVariable(
+                    Variable.builder().setName("matchIfMissing").setType(TypeNode.BOOLEAN).build()))
+            .setValueExpr(
+                ValueExpr.withValue(
+                    PrimitiveValue.builder().setValue("false").setType(TypeNode.BOOLEAN).build()))
+            .build();
     AnnotationNode conditionalOnPropertyNode =
         AnnotationNode.builder()
             .setType(types.get("ConditionalOnProperty"))
-            .setDescription(
-                "value = \""
-                    + Utils.springPropertyPrefix(libName, service.name())
-                    + ".enabled\", matchIfMissing = false")
+            .addDescription(valueStringAssignmentExpr)
+            .addDescription(matchIfMissingAssignmentExpr)
+            .build();
+
+    TypeNode clazzType =
```

Comment:
I believe this TypeNode is also defined in [dynamic types](https://github.com/googleapis/gapic-generator-java/blob/62cd6384e2afa58ee7b2f8d6c452f15e0ac05223/src/main/java/com/google/api/generator/spring/composer/SpringAutoConfigClassComposer.java#L87). see details in comment below.